pkg/storage/filepath/jsonfile.go

```diff
@@ -14,24 +15,27 @@ import (
 //   - For namespaced-scoped resources: the resource will be written under the root-path in
 //     the following structure:
 //
-//       -- (root-path) --- /namespace1/ --- resource1
-//                      |                |
-//                      |                --- resource2
-//                      |
-//                      --- /namespace2/ --- resource3
+//     -- (root-path) --- /namespace1/ --- resource1
+//     |                |
+//     |                --- resource2
+//     |
+//     --- /namespace2/ --- resource3
+//
```

Comment:
This is gofmt being too eager, I will revert these two comment changes.

Reply:
lol